com.microsoft.mrtk.graphicstools.unity/Samples~/UnityUI/UnityUI.unity

```diff
@@ -193,24 +193,25 @@ MonoBehaviour:
   _NormalMapScale: 1
```

Comment:
After loading the project in Unity 2020 some of the materials look off? 
![image](https://user-images.githubusercontent.com/13305729/207734042-aa07effb-c2b9-40a5-94fe-400800f22a5b.png)

I've seen this happen when going from 2021 to 2020 since Unity change how material keywords are stored. You might need to fix u a few materials.